eng/doc/CrossPlatformCryptography.md

```diff
@@ -79,83 +85,88 @@ This section includes the following packages:
 
 This section includes the following packages:
 
-* [crypto/aes](https://pkg.go.dev/crypto/aes)
-* [crypto/cipher](https://pkg.go.dev/crypto/cipher)
-* [crypto/des](https://pkg.go.dev/crypto/des)
-* [crypto/rc4](https://pkg.go.dev/crypto/rc4)
-
-| Cipher + Mode | Windows  | Linux            |
-|---------------|----------|------------------|
-| AES-ECB       | ✔️       | ✔️              |
-| AES-CBC       | ✔️       | ✔️              |
-| AES-CTR       | ❌       | ✔️              |
-| AES-CFB       | ❌       | ❌              |
-| AES-OFB       | ❌       | ❌              |
-| AES-GCM       | ✔️       | ✔️              |
-| DES-CBC       | ✔️       | ⚠️<sup>1</sup>  |
-| DES-ECB       | ✔️       | ⚠️<sup>1</sup>  |
-| 3DES-ECB      | ✔️       | ✔️              |
-| 3DES-CBC      | ✔️       | ✔️              |
-| RC4           | ✔️       | ⚠️<sup>1</sup>  |
+- [crypto/aes](https://pkg.go.dev/crypto/aes)
+- [crypto/cipher](https://pkg.go.dev/crypto/cipher)
+- [crypto/des](https://pkg.go.dev/crypto/des)
+- [crypto/rc4](https://pkg.go.dev/crypto/rc4)
+
+| Cipher + Mode | Windows | Linux          | macOS |
+| ------------- | ------- | -------------- | ----- |
+| AES-ECB       | ✔️      | ✔️             | ✔️    |
+| AES-CBC       | ✔️      | ✔️             | ✔️    |
+| AES-CTR       | ❌      | ✔️             | ❌    |
+| AES-CFB       | ❌      | ❌             | ❌    |
+| AES-OFB       | ❌      | ❌             | ❌    |
+| AES-GCM       | ✔️      | ✔️             | ✔️    |
+| DES-CBC       | ✔️      | ⚠️<sup>1</sup> | ✔️    |
+| DES-ECB       | ✔️      | ⚠️<sup>1</sup> | ✔️    |
+| 3DES-ECB      | ✔️      | ✔️             | ✔️    |
+| 3DES-CBC      | ✔️      | ✔️             | ✔️    |
+| RC4           | ✔️      | ⚠️<sup>1</sup> | ✔️    |
 
 <sup>1</sup>When using OpenSSL 3, requires the legacy provider to be enabled.
 
 ### AES-GCM keys, nonces, and tags
 
-* Key Sizes
+- Key Sizes
 
   AES-GCM works with 128, 192, and 256-bit keys.
 
-* Nonce Sizes
+- Nonce Sizes
 
   AES-GCM works with 12-byte nonces.
 
-* Tag Sizes
-  
+- Tag Sizes
+
   AES-GCM works with 16-byte tags.
 
 ## Asymmetric encryption
 
 This section includes the following subsections:
 
-* [RSA](#rsa)
-* [ECDSA](#ecdsa)
-* [ECDH](#ecdh)
-* [Ed25519](#ed25519)
-* [DSA](#dsa)
+- [RSA](#rsa)
+- [ECDSA](#ecdsa)
+- [ECDH](#ecdh)
+- [Ed25519](#ed25519)
+- [DSA](#dsa)
 
 ### RSA
 
+> [!WARNING]  
+> System libraries might not fully implement all RSA features. Please see the subsections below for more information.
```

Comment:
Why a warning for RSA in particular? Many of the tables have feature gaps explained below them.

Reply:
hmm good point, maybe it's implied (or perhaps we should have this warning at the top of the document)?

Reply:
It sounds like the main concern here is that it's not necessarily clear that there's more info below the table--in that case I think we use numbered notes more consistently and bring global-ish info above the table:
* https://github.com/microsoft/go/pull/1507

Reply:
More generally, I think these parts establish pretty well that features have limitations:

> At a fine-grained level, Go apps will fall back to the native Go implementation of an algorithm if the OS libraries don't support it. This article identifies the features that are supported on each platform.

> When an algorithm is not supported or the limitations are exceeded, Microsoft Go will fall back to the Go implementation.